pages/design/ui-patterns/progressive-disclosure.mdx

```diff
@@ -10,10 +10,10 @@ The following table outlines common progressive disclosure solutions in use at G
 
 | Component | Icon | Usage | Notes |
 | --- | --- | --- | --- |
-| Chevron | ![chevron](https://user-images.githubusercontent.com/24916540/52089727-1ae9d480-2564-11e9-8291-1ee9d3225427.png) | When elements of content are collapsed and can be toggled open | Do not use this icon to trigger a dropdown menus or navigation; the Caret icon is the more suitable for this. | 
-Expand/ Collapse Icon | ![expand](https://user-images.githubusercontent.com/24916540/52089724-1a513e00-2564-11e9-84f8-eba077f6abfc.png) | Signify that there is content (typically text) to be revealed | Should generally stand alone, rather than being paired with text | 
-Ellipses Icon | ![ellipses](https://user-images.githubusercontent.com/24916540/52089725-1ae9d480-2564-11e9-8d29-1dc28740924a.png) | For toggling truncated inline text content | Do not use this icon to trigger a dropdown menus or navigation; the Kebab icon is the more suitable for this. | 
-Text-only Toggles | | | Usage of this solution is discouraged, as generally icons or icon+text pairings provide better accessibility and more information |
+| Chevron | ![chevron](https://user-images.githubusercontent.com/24916540/52089727-1ae9d480-2564-11e9-8291-1ee9d3225427.png) | When elements of content are collapsed and can be toggled open | Do not use this icon to trigger a dropdown menus or navigation; the Caret icon is the more suitable for this. |
+Expand/ Collapse Icon | ![expand](https://user-images.githubusercontent.com/24916540/52089724-1a513e00-2564-11e9-84f8-eba077f6abfc.png) | Signify that there is content (typically text) to be revealed | Should generally stand alone, rather than being paired with text |
+Ellipses Icon | ![ellipses](https://user-images.githubusercontent.com/24916540/52089725-1ae9d480-2564-11e9-8d29-1dc28740924a.png) | For toggling truncated inline text content | Do not use this icon to trigger a dropdown menus or navigation; the Kebab icon is the more suitable for this. |
+Text-only Toggles | [example]() |  | Usage of this solution is discouraged, as generally icons or icon+text pairings provide better accessibility and more information |
```

Comment:
I don't know if we need to include this example as it doesn't fully represent the full interaction. Maybe we could show a bit more to get more context?

Reply:
I think it would be great if we could show something like [what Emily suggested](https://www.dropbox.com/s/gq5di6wy11fbi62/Screenshot%202019-06-26%2011.31.35.png?dl=0) in this section.